test/potluck_test.rb

```diff
@@ -0,0 +1,64 @@
+require './lib/potluck.rb'
+require 'minitest/autorun'
+require 'minitest/pride'
+require 'pry'
+
+class PotluckTest < Minitest::Test
+
+  def test_it_exists
+    potluck = Potluck.new("7-13-18")
+    assert_instance_of Potluck, potluck
+  end
+
+  def test_it_has_date
+    potluck = Potluck.new("7-13-18")
+    assert_equal "7-13-18", potluck.date
+  end
+
+  def test_it_has_dishes
+    potluck = Potluck.new("7-13-18")
+    assert_equal [], potluck.dishes
+  end
+
+  def test_it_can_add_dish
+    potluck = Potluck.new("7-13-18")
+    assert_equal ["Couscous Salad"], potluck.add_dish("Couscous Salad")
+    assert_equal ["Couscous Salad", "Cocktail Meatballs"], potluck.add_dish("Cocktail Meatballs")
```

Comment:
This works, and might be the easiest thing to test. As long as you understand that the intended interaction pattern is to pass objects (like you do in the test on lines 30 - 40) and not strings.